### ISSUE TITLE
Analytics Implementation or QA Support for Profile - Hub

### ISSUE THREAD

**Issue by mtcA6** [department-of-veterans-affairs/va.gov-team#72339]
# Google Analytics Implementation Request

### Please provide the following details for this request

- AE Profile 
- Slack `#accountexp-authexp`
- Profile Hub

- **When is the launch date?**
    - 12/18/2023

- **Provide the link(s) for Product/Page urls**
  
    -  staging.va.gov/profile

- **What Events need to be tracked?** _NOTE: [Design System Components](https://depo-platform-documentation.scrollhelp.site/analytics-monitoring/Design-System-Components.1840808040.html) are automatically tracked and should be used whenever possible to avoid custom configuration_

### Profile Hub event background 
- We have tracking on the profile hub clicks, that was setup with:
     - #63551

### Profile Hub Problem
- The problem is, I missed a couple of links and are now firing in an unexpected way. 

When a user clicks the list of events below, it first fires `Navigation - Link Click - Profile Hub -` and then it also triggers a separate link click event:

- Navigation - Link Click - Learn how to verify your identity on VA.gov
- Navigation - Link Click - Learn how to request your DD214 and other military records
- Navigation - Link Click - View payment history
- Navigation - Link Click - Update your sign-in info on the My HealtheVet website
- Navigation - Link Click - Update your sign-in info on the Login.gov website
- Navigation - Link Click - Update your sign-in info on the ID.me website
- Navigation - Link Click - Update your sign-in info on the DS Logon website

I think what we need is for the events listed above to also have "profile hub" in the naming convention so it should look something like this: 

- Navigation - Link Click - Profile Hub - Learn how to verify your identity on VA.gov
- Navigation - Link Click - Profile Hub - Learn how to request your DD214 and other military records
- Navigation - Link Click - Profile Hub - View payment history
- Navigation - Link Click - Profile Hub - Update your sign-in info on the My HealtheVet website
- Navigation - Link Click - Profile Hub - Update your sign-in info on the Login.gov website
- Navigation - Link Click - Profile Hub - Update your sign-in info on the ID.me website
- Navigation - Link Click - Profile Hub - Update your sign-in info on the DS Logon website

### Test users
- **If applicable, what test users are needed?**  _NOTE: Do not put staging credentials in this ticket; store or reference them in a .md file in the `va.gov-team-sensitive` repository_

    - any LOA3 test user can be used to check with an unblocked profile

### Any additional comments
-  I've created [this report ](https://analytics.google.com/analytics/web/?authuser=0#/analysis/a50123418w177519031p176188361/edit/_K1zzMjpQ2qxlfcC--aNiw) where we can see all of the clicks and essentially I'm hoping after we adjust things this row ` Navigation - Link Click - Profile Hub -` will disappear

> Please leave the following blank

## Acceptance criteria
- [ ] All appropriate issue tagging is completed
- [ ] QA has been completed
- [ ] Events have been added to `Product` and `Benefit Hub` Content Groups

## Additional information

Please refer to [Platform Collaboration Cycle](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/index.html) or the [Analytics Request Touchpoint](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Analytics-request.1782120453.html) on Platform website for more information about the Collaboration Cycle.

**Comment by mtcA6** [comment]
cc: @Samara-Strauss

**Comment by michelle-dooley** [comment]
@mtcA6 the team will review and get back to you with timing.

**Comment by mtcA6** [comment]
I'm going to close this out as we have a ticket #75426 to complete the work necessary to correct the issues we're seeing. If we're still seeing issues once the work is completed, I'll either open a new ticket or will reopen this one.
